integration_tests/integration/analytics.spec.ts

```diff
@@ -316,10 +373,9 @@ context('Analytics section', () => {
     it('users see analytics', () => {
```

Comment:
Decided to only test the default protected characteristics page (age) because the other characteristics do not have any specific logic in the template